src/utils/echo.util.js

```diff
@@ -0,0 +1,13 @@
+const EchoUtil = {
+  echo(receiverId, data, connectedClients, io, type) {
+    if (!receiverId) {
+      io.emit(type, data);
+    } else if (connectedClients[receiverId.toString()]) {
+      connectedClients[receiverId.toString()].forEach(element => {
+        io.to(element).emit(type, data);
+      });
+    }
+  }
+};
+
+export default EchoUtil;
```

Comment:
Awesome! this makes it universal

Reply:
- Move this method to the `ChatService` and reference it inside the `CommentsController` and `ChatsController`.
- The name `echo` does not seem descriptive enough. Perhaps `sendChatNotification` or something like that.